src/category_theory/limits/has_limits.lean

```diff
@@ -90,22 +91,32 @@ variables (J C)
 class has_limits_of_shape : Prop :=
 (has_limit : Π F : J ⥤ C, has_limit F . tactic.apply_instance)
 
-/-- `C` has all (small) limits if it has limits of every shape. -/
-class has_limits : Prop :=
+/--
+`C` has all limits of size `v u` (`has_limits_of_size.{v u} C`)
+if it has limits of every shape `J : Type u` with `[category.{v} J]`.
+-/
+class has_limits_of_size : Prop :=
```

Comment:
Would it make sense to restate the `(C : Type u) [category.{v} C]` explicitly in this definition, so that people reading through this file have all the universe families visible in this declaration?